packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,279 @@
+
+Persona is used in PeoplePicker, Team's left rail and menus, chiclets, and Card.
+
+_**Persona Figma spec:**_ [link](https://www.figma.com/file/ayf1r31NnONkfiE00g9QOv/Persona?node-id=1741%3A2221)
 
 ## Prior Art
 
-_Include background research done for this component_
+- OpenUI research: [Avatar/Persona](https://open-ui.org/components/avatar.research)
+- v7/v8: [link](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)
+  - > Note: v8's Persona is a combination of v9's Persona and Avatar (v8's Persona has a sub-component PersonaCoin that would count as Avatar, but is not exposed.). In v9 we've taken the approach of having a single component Avatar and another component Persona.
+- v0: Does not have equivalent component.
+
+### Coin
+
+_Avatar vs PersonaCoin_: As mentioned in Prior Art, v8's Persona is not quite the same as v9's. v8's Persona is only able to showcase PersonaCoin while v9 showcases Avatar (including Avatar + PresenceBadge, Avatar + image, and Avatar + Icon). Note that PersonaCoin does not support custom icons, the only icon options are the size 8 icon shown below and the unknown PersonaCoin.
+
+| v8 PersonaCoin ([link](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)) | v9 Avatar ([link](https://react.fluentui.dev/?path=/docs/components-avatar--default)) |
+| :-------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------: |
+|                             ![](https://i.imgur.com/TuUXN7t.png)                              |                         ![](https://i.imgur.com/BbEhlc3.png)                          |
+|                             ![](https://i.imgur.com/pXEFFmu.png)                              |                         ![](https://i.imgur.com/hbms9Bp.png)                          |
+
+_Icon vs PersonaCoin_: PersonaCoin allows you to have icon only when there's no image available or it's the unknown Persona coin variant. In v9's Persona, you have the option of showcasing an icon through the Avatar or showcasing only the icon.
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- Note: The icon only shows up if there's no presence and size is `tiny`, `size8`, or `size16`.
+
+_Image vs PersonaCoin_: PersonaCoin allows you to have an image only within the Avatar. This can be avoided by rendering a custom coin, but there's still all the dom and styling overhead that comes with PersonaCoin. v9's Persona allows you use a single image with no extra dom.
+
+_PresenceBadge vs PersonaCoin_: PersonaCoin allows you to have only a PresenceBadge when the size of the coin is `tiny`, `size8`, or `size16`. In v9's Persona you are able to render the PresenceBadge alone.
+
+### Text lines
+
+Other than styling and naming, the text lines in Persona remain the same.
 
```

Comment:
It would be great to include a section with a component anatomy as it starts with sample code, but we don't know how it looks 🤔